src/Text/Trifecta/Pretty.hs

```diff
@@ -0,0 +1,57 @@
+{-# LANGUAGE CPP #-}
+-----------------------------------------------------------------------------
+-- |
+-- Copyright   :  (C) 2011-2015 Edward Kmett
+-- License     :  BSD-style (see the file LICENSE)
+--
+-- Maintainer  :  Edward Kmett <ekmett@gmail.com>
+-- Stability   :  experimental
+-- Portability :  non-portable
+--
+-- An interface between the prettyprinter and the ANSI SGR primitives
+-- provided by the ansi-terminal package.
+----------------------------------------------------------------------------
+
+module Text.Trifecta.Pretty
+  ( ANSIPretty (..)
+  , AnsiStyle
+  , renderIO
+  -- * Rendering
+  , char
+  -- * Styles
+  , bold
+  , debold
+  , underlined
+  , deunderline
+  -- * Compatibility shims
+  , renderPretty
+  , columns
+  ) where
+
+
+#if __GLASGOW_HASKELL__ < 710
+import Data.Monoid
+#endif
+
+import Data.Text.Prettyprint.Doc
+import Data.Text.Prettyprint.Doc.Render.Terminal
+import Data.Text.Prettyprint.Doc.Render.Terminal.Internal (ansiBold, ansiUnderlining)
+
+class ANSIPretty a where
```

Comment:
I see. When I thought about the migration my first idea that this is fishy. I'd rather have `prettyDelta`, `prettyHighlightedRope` etc. There are just a few, and AFAIK we never abstract over them (i.e. we don't have `AnsiPretty a => ...` functions).